bin/node/src/commands/bundled.rs

```diff
@@ -82,6 +84,15 @@ pub enum BundledCommand {
             value_name = "DURATION"
         )]
         grpc_timeout: Duration,
+
+        /// Filepath to the insecure validator secret key for signing blocks.
+        ///
+        /// Only used in development and testing environments.
+        #[arg(
+            long = "validator-secret-key-filepath",
+            value_name = "VALIDATOR_SECRET_KEY_FILEPATH"
+        )]
+        validator_secret_key_filepath: Option<PathBuf>,
```

Comment:
Related to the previous comment, I'm not sure this needs to be a filepath. A secret key is just a 32-byte string that could be hex encoded. But maybe keeping it as a file simplifies things?

Reply:
Given that this is only intended for devnet/local testing I think taking it inline is good enough.

What we can do is give it a more specific name e.g.

```
validator.development.secret_key
```